src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -129,6 +136,21 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        private void CheckBackwardCompatibility(ref bool disableAtomicDocumentWrites)
+        {
+            if (disableAtomicDocumentWrites) 
+                return;
+
+            if (RequestRouter.TryGetClientVersion(HttpContext, out var clientVersion) == false)
+            {
+                disableAtomicDocumentWrites = true;
+                return;
+            }
+
+            if (clientVersion.Major <= 5 && clientVersion.Minor < 2)
```

Comment:
`clientVersion.Major < 5 || (clientVersion.Major == 5 && clientVersion.Minor < 2)`

would be the most appropriate one?

Reply:
done